js/src/components/contact-information/phone-number-card/useCountdown.test.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * External dependencies
+ */
+import { renderHook, act } from '@testing-library/react-hooks';
+
+/**
+ * Internal dependencies
+ */
+import useCountdown from './useCountdown';
+
+describe( 'useCountdown', () => {
+	it( 'initially should return `{ second: 0, callCount: 0, startCountdown: Function }`', () => {
+		const { result } = renderHook( () => useCountdown( 'A' ) );
+
+		expect( result.current.second ).toBe( 0 );
+		expect( result.current.callCount ).toBe( 0 );
+		expect( result.current.startCountdown ).toBeInstanceOf( Function );
+	} );
+
+	it( 'should record the calling count of `startCountdown()`', () => {
+		const { result } = renderHook( () => useCountdown( 'A' ) );
+
+		act( () => {
+			result.current.startCountdown( 10 );
+		} );
+
+		expect( result.current.callCount ).toBe( 1 );
+
+		act( () => {
+			result.current.startCountdown( 20 );
+		} );
+
+		expect( result.current.callCount ).toBe( 2 );
+
+		act( () => {
+			result.current.startCountdown( 30 );
+		} );
+
+		expect( result.current.callCount ).toBe( 3 );
+	} );
+
+	describe( 'For the countdowns updated by interval timers', () => {
+		let getTime;
+
+		beforeEach( () => {
+			jest.useFakeTimers();
+			getTime = jest
+				.spyOn( Date.prototype, 'getTime' )
+				.mockName( 'Date.getTime' );
+		} );
+
+		afterEach( () => {
+			jest.clearAllTimers();
+			jest.useRealTimers();
+		} );
+
+		it( 'should start countdown 10 to 0 by calling `startCountdown( 10 )` and update the current remaining countdown to `second` value after every second.', () => {
```

Comment:
:nail_care: 
I'm not an English expert, but shouldn't it be more like:
```suggestion
		it( 'should start countdown 10 to 0 by calling `startCountdown( 10 )` and update the `second` property every second with the current remaining countdown.', () => {
```